### ISSUE TITLE
Crash when opening the blocks UI

### ISSUE THREAD

**Issue by xZalix** [MichaelHillcox/XRay-Mod#166]
Everytime i try to open the ui to add my modded ores the game crashes
My minecraft is 1.12.2 and the xray version is 1.5.0

here is my latest log
[latest.log](https://github.com/MichaelHillcox/XRay-Mod/files/5672397/latest.log)

**Comment by MichaelHillcox** [comment]
Try updating to 1.6.0.

**Comment by xZalix** [comment]
> Try updating to 1.6.0.

apologies for late response,
i tried to update it and it kept crashing when starting the game
here is a log 
[crash-2020-12-15_07.11.40-client.txt](https://github.com/MichaelHillcox/XRay-Mod/files/5692317/crash-2020-12-15_07.11.40-client.txt)

**Comment by MichaelHillcox** [comment]
Delete your config/Xray folder and try again

**Comment by ilikeminecraft811** [comment]
Hello, I am trying to use this mod with another mod called the legends mod which has its own ores. However when I tried to add a specific ore from the legends mod to the x ray mod, it would just crash my game. I tried looking at the block and holding it in my hand but no matter what I do it would just crash. I even tried to add stone to the x ray mod and it would still crash. If I can have some help or an explanation to this I would be very grateful.

I am using the minecraft version 1.7.10
